Tools/make-pkgconfig.sh

```diff
@@ -1,14 +1,26 @@
 #!/usr/bin/env bash
 
 version=$(llvm-config --version)
-filename=/usr/local/lib/pkgconfig/llvm.pc
+filename=$1
 
 mkdir -p `dirname $filename`
 touch $filename
 
+# REVISIT(nickpdemarco):
+# Why does macos need the standard library explicitly linked, while linux does not?
+# This does not feel like the correct place for this logic.
```

Comment:
In fact I'm surprised linux doesn't require the standard library too.

I suspect clang on macOS might behave differently because, by default, the Apple toolchain will expect to build Objective-C. I'm only guessing, though.